### PR TITLE
[TempusController] return redeemed YBT/BT amounts

### PR DESCRIPTION
**Pull request by yuval-weiss**

### CODE REVIEW

File: contracts/TempusController.sol [hunk 1]
```diff
@@ -638,13 +646,13 @@ contract TempusController is ITempusController, ReentrancyGuard, Ownable, Versio
...
             if (((yields > principals) ? (yields - principals) : (principals - yields)) >= maxLeftoverShares) {
-                (uint256 swapAmount, IPoolShare tokenIn) = tempusAMM.getSwapAmountToEndWithEqualShares(
+                (uint256 swapAmount, ) = tempusAMM.getSwapAmountToEndWithEqualShares(
                     principals,
                     yields,
                     maxLeftoverShares
                 );
 
-                bool yieldsIn = tokenIn == tempusPool.yieldShare();
+                bool yieldsIn = yields > principals;
```
[the remark this answers is not in the document]
Comment (yuval-weiss) [reply contracts/TempusController.sol:655]: I was getting a `Stack too deep` compilation error...

File: contracts/ITempusController.sol [hunk 2]
```diff
@@ -199,6 +201,7 @@ interface ITempusController {
     /// @param yieldsStaked Amount of staked yields (in TempusAMM) to redeem
     /// @param maxLpTokensToRedeem Maximum amount of LP tokens to spend for staked shares redemption
     /// @param toBackingToken If true redeems to backing token, otherwise redeems to yield bearing
+    /// @return Amount of Yield Bearing Tokens (if `toBackingToken == false`) or Backing Tokens (if `toBackingToken == true`) that were imbursed as a result of the redemption
```
Comment (RedFox20) [inline contracts/ITempusController.sol:204]: Line length issue

File: contracts/TempusController.sol (hunk 1 above)
[the remark this answers is not in the document]
Comment (RedFox20) [reply contracts/TempusController.sol:655]: Agree, this can be reused outside the `if`.

[the remark this answers is not in the document]
Comment (yuval-weiss) [reply contracts/TempusController.sol]: Ah right, thanks :)

File: contracts/ITempusController.sol (hunk 2 above)
Comment (yuval-weiss) [reply contracts/ITempusController.sol]: whoops, fixed!

**Review by RedFox20 — APPROVED** [review]

**Pull request merged**